src/main.rs

```diff
@@ -149,13 +149,25 @@ struct Options {
     /// Utility log level
     ///
     /// Set to `debug` when submitting a bug report.
+    /// Set to `info` to only print useful information.
+    /// Set to `Warn` to only print on hazardous situations.
+    /// Set to `Error` to only print serious errors.
+    ///
+    /// Set to `off` to disable logging completely, this will also
+    /// disable output from `cargo-install`.
     #[clap(
         help_heading = "Meta",
         long,
         default_value = "info",
         value_name = "LEVEL"
     )]
     log_level: LevelFilter,
+
+    /// Equivalent to setting `log_level` to `off`.
+    ///
+    /// This would override the `log_level`.
+    #[clap(help_heading = "Meta", long)]
+    quiet: bool,
```

Comment:
For compat with cargo-install, let's also use the short `-q` for it.

Reply:
Done!